tarantool_test.go

```diff
@@ -2931,6 +2811,22 @@ func TestStream_DoWithClosedConn(t *testing.T) {
 	}
 }
 
+func TestConnectionBoxSessionPushUnsupported(t *testing.T) {
+	old := log.Writer()
+	defer log.SetOutput(old)
+	var buf bytes.Buffer
+	log.SetOutput(&buf)
+
+	conn := test_helpers.ConnectWithValidation(t, dialer, opts)
+	defer conn.Close()
+	_, err := conn.Do(NewCallRequest("push_func").Args([]interface{}{1})).Get()
+	require.NoError(t, err)
+
+	actualLog := buf.String()
+	expectedLog := "unsupported box.session.push()"
+	require.Contains(t, actualLog, expectedLog)
+}
```

Comment:
Just a code-style issues:
```suggestion
func TestConnectionBoxSessionPushUnsupported(t *testing.T) {
	old := log.Writer()
	defer log.SetOutput(old)

	var buf bytes.Buffer
	log.SetOutput(&buf)

	conn := test_helpers.ConnectWithValidation(t, dialer, opts)
	defer conn.Close()

	_, err := conn.Do(NewCallRequest("push_func").Args([]interface{}{1})).Get()
	require.NoError(t, err)

	actualLog := buf.String()
	expectedLog := "unsupported box.session.push()"
	require.Contains(t, actualLog, expectedLog)
}
```